docs/data_structures.md

```diff
@@ -0,0 +1,74 @@
+
+## Tasks
+To create a new mapping task, the overall project extent is split up into many single tasks.
+Tasks are the smallest unit in the MapSwipe data model.
+
+### For built area projects
+They are derived from the area of interest by gridding it into many small equal-sized rectangular polygons.
+Each task corresponds to a specific tile coordinate from a tile map service (TMS) using a web Mercator projection as its geographical reference system.
+Therefore, each task is characterized by a geometry and its tile coordinates, which describe its x, y and z position.
+For the projects analysed in this project, the tiles for all tasks are generated at zoom level 18.
+Taking the latitude of each task location into account the satellite imagery has a maximum spatial resolution of ~ 0.6 meter at the equator.
+
+### For building footprint validation projects
+
+A task is the validation of the footprint of a single building. This would be a single "way" in OSM terminology. With the data used in input, multipolygons shouldn't appear, and if they do, they will be ignored here. Similarly, a single node will be ignored.
+The data required to complete a task includes:
+- the building geometry: 
+
+| Parameter | Description | Example: Built areas | Example: Building footprint |
+| --- | --- | --- | --- |
+| **Id** | Each task can be identified by its Id. The Id is a composition of its position in the corresponding tile map system, which can be described by the x, y and z coordinates. | "18-160175-127032" |  |
+| **Project Id** | The Id of the project this task relates to | "1" |  |
+| **Tile Z** | The z coordinate of the tile defines the zoom level. Greater values for z will correspond to higher spatial resolution of the corresponding image. For most regions Bing provides images up to zoom level 18. For aerial imagery or images captured by UAVs even higher z values are valid. | "18" | Similar to existing projects: These can be determined by the worker scripts, so the app has little to do here. |
+| **Tile X**| The x coordinate characterises the longitudinal position of the tile in the overall tile map system taken the zoom level into account. The x coordinates increase from west to east starting at a longitude of -180 degrees. | "160175" | See above |
+| **Tile Y** | The y coordinate characterises the latitudinal position of the tile in the overall tile map system taken the zoom level into account. The latitude is clipped to range from circa -85 to 85 degrees. The y coordinates increase from north to south starting at a latitude of around 85 degrees. | "127032" | See above |
+| **Footprint** | The list of nodes constituting the footprint corners. There must be at least 3 nodes. | Not used | [(x1, y1), (x2, y2), (x3, y3), (x4, y4)] |
```

Comment:
Instead of "footprint" I would still name it "geometry". We used this already for built_area projects. (Now, we don't store the geometries for these tasks anymore, because we don't need it in the client)

We might want to use a binary format such as WKB to store geometries. We need to make sure that the size (size in MB etc.) of a group does not get too big. This will be costly.